template_config.yml

```diff
@@ -5,15 +5,15 @@
 
 additional_repos:
 - bindings: true
-  branch: 0.13
+  branch: 0.13.2
   name: pulp_ansible
   org: pulp
 - bindings: true
-  branch: 2.10.2
+  branch: 2.10.5
   name: pulp_container
   org: pulp
 - bindings: false
-  branch: v0.4.2
+  branch: v0.4.5
```

Comment:
You also need to update `pulpcore_branch: 3.18.1` below in this file

Reply:
Updated.